### DISCUSSION TITLE
Is there a way to dynamically set the colour and weight of a scatter line?

### DISCUSSION THREAD

**Discussion by howick19** [LayTec-AG/Plotly.Blazor#319] [Q&A] [answered]
Was trying to dynamically set the weight/width and colour of a scatter line (customer wanted the average to be red and a thicker line), but couldn't find how to do this in any examples.  Is it possible?

Thanks

**Comment by damian-clarke-wearemci** [comment, marked as answer]
Try line property of ScatterGl, i.e

Line = new Plotly.Blazor.Traces.ScatterGlLib.Line {Color = "black", Width = 5 }

**Reply by howick19** [reply]
That works - thanks so much! :-)
